modules/RTC/CodecSelection.js

```diff
@@ -127,6 +131,83 @@ export class CodecSelection {
         return supportedCodecs;
     }
 
+    /**
+     * Processes the encode time stats received for all the local video sources and switches the codec to a low
```

Comment:
We don't do any codec change here yet AFAICT, pl update the comment.

Reply:
Ah right, I didn't add the switching logic in this PR since it was getting long, forgot to update the comment.